docs/examples/201/cdn-with-storage-account/main.bicep

```diff
@@ -0,0 +1,57 @@
+param location string {
+  default: resourceGroup().location
+}
+
+var storageAccountName = format('{0}{1}', 'storage', uniqueString(resourceGroup().id))
+var endPointName = format('{0}{1}', 'endpoint-', uniqueString(resourceGroup().id))
+var profileName = 'CdnProfile1'
+var storageAccountHostName = replace(replace(reference(storageAccountName).primaryEndpoints.blob, 'https://', ''), '/', '')
```

Comment:
change `reference(storageAccountName).primaryEndpoints.blob` to `storageAccount.properties.primaryEndpoints.blob`